Week-1/Assignment 1.2/style.css

```diff
@@ -0,0 +1,183 @@
+body {font-family: "Lato", sans-serif;}
+
+i{
+    margin-bottom: 12px;
+}
+
+:root {
```

Comment:
Good work. This can further be extended to something like the below: 

```
/* Font families. */
--font-family-sans: "Roboto", sans-serif;
--font-family-serif: "Playfair Display", serif;

/* Font sizes. */
--font-size-sm: 0.875rem;
--font-size-md: 1rem;
--font-size-lg: 1.125rem;
--font-size-xl: 1.25rem;
--font-size-2xl: 1.5rem;
--font-size-3xl: 2rem;
--font-size-4xl: 3rem;

/* Line heights. */
--line-height-sm: 1.25;
--line-height-md: 1.5;

/* Letter spacings. */
--letter-spacing-sm: 0.05em;
--letter-spacing-md: 0.1em;
```

Reply:
Thanks for suggestions will keep in mind now onwards.